task8/src/main/java/AdsServlet.java

```diff
@@ -0,0 +1,57 @@
+import adList.Ad;
```

Comment:
Java package is missing. It's considered necessary to place your Java code within some package. Usually the package starts with the same words you use as the artifact ID, and then come more folders describing the nature of code within. So, for a servlet, this would be like `org.example.servlets`, for a utility class - `org.example.util`, etc.